src/adminApi.js

```diff
@@ -95,12 +96,12 @@ function getPaginatedJson(uri) {
         if (!json.data) return json;
         if (!json.next) return json.data;
 
-        if (json.data.length < 100) {
+        if (json.data.length < pageSize) {
             // FIXME an hopeful hack to prevent a loop
             return json.data;
         }
 
-        return getPaginatedJson(json.next).then(data => json.data.concat(data));
+        return getPaginatedJson(json.next, null).then(data => json.data.concat(data));
```

Comment:
I'd expect this param to be `pageSize` rather than `null`.  If you're leaving it null because paging doesn't even work in our environment I'd add a comment explaining it.

Reply:
..or maybe it doesn't matter since paging is embedded in the next url, so nvm

Reply:
Yeah, `json.next` already contains the paging params.